static/src/server.ts

```diff
@@ -465,9 +465,9 @@ app.get("/nodejs/query", (req: Request, res: Response) => {
   const urlRoot = `${req.protocol}://${req.get("host")}`;
   res.setHeader("Content-Type", "application/json");
   axios
-    // Use "udp=false" to disable using default place.
+    // Use "mode=strict" to disable using default place, use a higher SV threshold, etc.
     .post(
-      `${CONFIG.apiRoot}/api/nl/data?q=${query}&detector=heuristic&udp=false&mode=strict`,
+      `${CONFIG.apiRoot}/api/nl/data?q=${query}&detector=heuristic&mode=strict`,
```

Comment:
nit: from the description in the pr, `mode=strict` also sets `detector=heuristic` right? so we can remove that param?